tests/core/daemon/test_daemon.py

```diff
@@ -1107,3 +1134,470 @@ async def test_keyring_file_deleted(
+            "x": False,
+            "b": 512,
+            "overrideK": True,
+            "delay": 0,
+            "a": 3598820529,
+            "c": "hello",
+        },
+        response={
+            "success": True,
+        },
+    ),
+    RouteCase(
+        route="start_plotting",
+        description="bladebit - diskplot",
+        request={
+            "service": "chia_plotter",
+            "plotter": "bladebit",
+            "plot_type": "diskplot",
+            "w": True,
+            "m": True,
+            "no_cpu_affinity": True,
+            "k": 25,
+            "r": 2,
+            "u": 128,
+            "e": False,
+            "parallel": False,
+            "n": 1,
+            "queue": "default",
+            "d": "root_path",
+            "t": "root_path",
+            "t2": "testing",
+            "f": "",
+            "plotNFTContractAddr": "",
+            "x": False,
+            "b": 512,
+            "overrideK": True,
+            "delay": 0,
+            "a": 3598820529,
+            "c": "hello",
+            "cache": "cache",
+            "f1_threads": 5,
+            "fp_threads": 6,
+            "c_threads": 4,
+            "p2_threads": 4,
+            "p3_threads": 4,
+            "alternate": True,
+            "no_t1_direct": True,
+            "no_t2_direct": True,
+        },
+        response={
+            "success": True,
+        },
+    ),
+    RouteCase(
+        route="start_plotting",
+        description="madmax",
+        request={
+            "service": "chia_plotter",
+            "plotter": "madmax",
+            "plot_type": "diskplot",
+            "w": True,
+            "m": True,
+            "no_cpu_affinity": True,
+            "k": 25,
+            "r": 2,
+            "u": 128,
+            "e": False,
+            "parallel": False,
+            "n": 1,
+            "queue": "default",
+            "d": "root_path",
+            "t": "root_path",
+            "t2": "testing",
+            "f": "",
+            "plotNFTContractAddr": "",
+            "x": False,
+            "b": 512,
+            "overrideK": True,
+            "delay": 0,
+            "a": 3598820529,
+            "c": "hello",
+            "v": 128,
+        },
+        response={
+            "success": True,
+        },
+    ),
+)
+@pytest.mark.asyncio
+async def test_plotter_options(
+    daemon_connection_and_temp_keychain: Tuple[aiohttp.ClientWebSocketResponse, Keychain],
+    get_b_tools: BlockTools,
+    case: RouteCase,
+) -> None:
+    ws, keychain = daemon_connection_and_temp_keychain
+
+    # register for chia_plotter events
+    service_name = "chia_plotter"
+    data = {"service": service_name}
+    payload = create_payload("register_service", data, "chia_plotter", "daemon")
+    await ws.send_str(payload)
+    response = await ws.receive()
+    assert_response_success_only(response)
+
+    case.request["t"] = str(get_b_tools.root_path)
+    case.request["d"] = str(get_b_tools.root_path)
```

Comment:
Or `tmp_path.joinpath("plotter_temp_dest")` or such from the `tmp_path` fixture?

Reply:
sure, although I think it's somewhat unimportant which tmp dir is used